instances/dev/main.tf

```diff
@@ -30,16 +30,31 @@ variable "perm_env" {
   }
 }
 
+resource "aws_autoscaling_group" "dev_api" {
+  name                      = "backend-dev"
+  max_size                  = 5 # does this seem appropriate?
```

Comment:
Previously this was set to a max of 3, but that was arbitrary! 5 seems like a a good end target, assuming we dial in appropriate scaling triggers.